Docs/Configuration.tex

```diff
@@ -1565,8 +1551,8 @@ \subsection{Quirks Properties}\label{booterpropsquirks}
   Reports security violation during attempts to write to \texttt{db}, \texttt{dbx},
   \texttt{PK}, and \texttt{KEK} variables from the operating system.
 
-  \emph{Note}: This quirk mainly attempts to avoid issues with NVRAM implementations
-  with problematic defragmentation, such as select Insyde or \texttt{MacPro5,1}.
+  \emph{Note}: This quirk attempts to avoid issues with NVRAM implementations with
+  fragmentation issues such as on the \texttt{MacPro5,1} as well as on certain Insyde firmware.
```

Comment:
Thank you!

Would suggest reinsert comma after "fragmentation issues"?

```
  This quirk attempts to avoid issues with NVRAM implementations with
  fragmentation issues, such as on the \texttt{MacPro5,1} as well as on certain Insyde firmware.
```